python/tank/platform/util.py

```diff
@@ -261,3 +261,33 @@ def get_logger(module_name):
     return logging.getLogger(full_log_path)
 
 
+def get_hyperlink_html(url, name, color=None, bold=True):
```

Comment:
Rather than adding it to the bundle base class, per discussion it's now implemented on the tk-framework-qtwidgets Framework object. It's then accessed from there in the qtwidgets framework and in the shotgunpanel app.